jsapp/js/components/processing/translations/stepSingleViewer.component.tsx

```diff
@@ -74,7 +78,10 @@ export default function StepSingleViewer(props: StepSingleViewerProps) {
             startIcon='edit'
             onClick={openEditor}
             tooltip={t('Edit')}
-            isDisabled={singleProcessingStore.data.isFetchingData}
+            isDisabled={
+              !hasManagePermissionsToCurrentAsset() ||
+              singleProcessingStore.data.isFetchingData
```

Comment:
Nitpicking, but please change order of these, so it's the same as in all the other places